test/truefi/LoanToken.test.ts

```diff
@@ -384,6 +384,80 @@ describe('LoanToken', () => {
     })
   })
 
+  describe('Reclaim', () => {
+    beforeEach(async () => {
+      await loanToken.fund()
+      await withdraw(borrower)
+      timeTravel(provider, monthInSeconds * 12)
+      await tusd.connect(borrower).approve(loanToken.address, parseEther('100'))
+    })
+
+    it('reverts when loan not closed', async () => {
+      await expect(loanToken.connect(borrower).reclaim())
+        .to.be.revertedWith('LoanToken: Current status should be Settled or Defaulted')
+    })
+
+    it('reverts when not borrower tries access', async () => {
+      await loanToken.close()
+      await expect(loanToken.reclaim())
+        .to.be.revertedWith('LoanToken: Caller is not the borrower')
+    })
+
+    it('reverts when total supply is greater than 0', async () => {
+      await loanToken.close()
+      await expect(loanToken.connect(borrower).reclaim())
+        .to.be.revertedWith('LoanToken: Cannot reclaim when LoanTokens in circulation')
+    })
+
+    it('reverts when balance is 0', async () => {
+      await loanToken.close()
+      await payback(borrower, parseEther('1100'))
+      await loanToken.redeem(parseEther('1100'))
+      await expect(loanToken.connect(borrower).reclaim())
+        .to.be.revertedWith('LoanToken: Cannot reclaim when balance 0')
+    })
+
+    const paybackRedeemPayback = async () => {
+      await payback(borrower, parseEther('900'))
+      await loanToken.redeem(parseEther('1100'))
+      await payback(borrower, parseEther('200'))
+    }
```

Comment:
Helpers definitions should be located at the beginning of `describe`, above all all `it`s

Reply:
Done